atd-vzv/src/views/summary/PeopleByDemographics.js

```diff
@@ -154,8 +154,20 @@ const PeopleByDemographics = () => {
         }
         return accumulator;
       }, 0);
-      const percentage = (categoryTotal / chartData[year].length) * 100;
-      return percentage;
+      const overallTotal = chartData[year].length;
+      const percentage = (categoryTotal / overallTotal) * 100;
+      const fullData = {
+        percentage: percentage,
+        categoryTotal: categoryTotal,
+        overallTotal: overallTotal
+      }
+      // If getFullData is true, return the fullData Object for display in tooltips,
+      // else return percentage for chartJS to render the chart
+      const data = getFullData ?
+      fullData
+      : percentage
+
+      return data
```

Comment:
It seems like prettier isn't enabled in your code editor. If it was, there would be a `;` at the ends of lines 168 and 170. Can you check and make sure? Its important for consistency that the code we push to `master` gets automatically code formatted. Let me or @mddilley know if you need some help.

Reply:
It's enabled, I just forgot to apply the formatting before pushing. Thanks for catching that - will be more diligent in the future!

Reply:
sound good! Rémy also had a suggestion to automate prettier/linting on the VZE app, maybe its worth porting over those npm scripts to this directory too.